artifacts/ts/Add.ts

```diff
@@ -41,6 +41,7 @@ import {
   Balances,
   MapValue,
   TokenBalance,
+  TupleTestStruct,
```

Comment:
It seems inefficient that whenever we add a new contract, it's imported into all other contracts' TS files.